docs/users/intro.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 1
+---
+# BobaBoard Basics
+
+_"BobaBoard will move you and fill you with a feeling of deep peace. I hope your time here will be pleasant and your memories will last forever."_
+
+## What is BobaBoard?
+**The Simple Answer:** A social media site made for fandom by people in fandom.
+
+**The Detailed Answer:** BobaBoard is the brain child of Ms. Boba who woke up to tumblr's porn ban and decided enough was enough. She set out to create an open source platform that could meet the evolving demands of fandom, a place that could support the needs of varying communities by being adaptable, stable, and fun to use, where fandom made the rules.
+
+The result is a social media site that's part-forum, part-Discord, part-LJ clone, and yet none of those at all.
+
+## How BobaBoard is Different...
+We hear you: you already have 37 social accounts and maybe use 3 of them, if you squint, why would you want another? Why give another startup platform a chance? What's _different_ about this place?
+
+### Realms
+:::info
+
+This section of the guide is still in development! We're still in the process of figuring out how Realms will work. The general idea is explained below, but your input will help shape things!
+
+:::
+
+BobaBoard is made up of distinct areas called **Realms**. Like LJ and its clones (and to some extent, Discord), each Realm is its own domain within the BobaBoard "network". Each Realm may have different goals, cultures, and rules.
+
+As BobaBoard expands to be open to the public, Moderators will be able to establish their own Realms. Users will have to join each Realm they want posting and reading access to and explicitly agree to their rules.
+
+There are currently two Realms making up BobaBoard:
+- V0: The Alpha
+  > The OG BobaBoard. A closed Alpha for Ms. Boba's friends, Boba volunteers, and early adopters.
+
+- Fandom Coders
+  > The first official "Realm". It's a community of fandom people who also code or want to learn to code, particularly if they use coding in their fandom activities!
+
+For more information on each Realm, visit their sections in the guide.
+
+### Boards, Not Blogs
```

Comment:
(coming back up here after reading the whole guide to leave a more general message about the structure) 

I feel like there's 2 big sections we might want to focus on:
1) BobaBoard's structure, with Realms & Boards
2) Posting on BobaBoard, with the identity system, tagging etc.

Reply:
I agree, and now I'm debating about having anything about anon identities here.

My general plan was for this to be a "Key Points" page, and I think anon identities are a key point to BobaBoard.

But I've got the things we talked about during our call sorted out into:

> __**SHARED STUFF**__
> - Development Status (and giving feedback?)
> - How to Use BobaBoard *all the UX stuff we mentioned that was important*
> - Shared Culture/Code/Rules *things like necroposting, OP has a say, don't be an asshole*
> 
> And then each Realm would get it's own section with more detailed info about rules, code of conduct, tagging guidelines, etc.

I wonder if I should just move the "Anonymous Identities" to the "How To" section since we were going to talk about the BobaDex there, and leave this page for the Boba structure info.